spec/fixtures/Weakly.podspec

```diff
@@ -8,6 +8,6 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '8.0'
   s.source       = { :git => "https://github.com/neonichu/CPDColors.git", :tag => s.version }
   s.source_files = 'Code'
-  
-  s.dependency 'PromiseKit'
+
+#  s.dependency 'PromiseKit'
```

Comment:
had to do cause I got the following in the console during running of this test case.

```
Pod::Informative: [!] Pods written in Swift can only be integrated as frameworks; this 
feature is still in beta. Add `use_frameworks!` to your Podfile or target to opt into using it.
	/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.1/lib/cocoapods/installer.rb:382:in `block (2 levels) in verify_framework_usage': Packager - can package a podspec with weak frameworks without strong linking
	/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.1/lib/cocoapods/installer.rb:378:in `each'
	/Library/Ruby/Gems/2.0.0/gems/cocoapods-0.37.1/lib/cocoapods/installer.rb:378
...
```

Appreciate any hint on a proper way to add this directive without causing side-effects on all other test-cases?